packages/bottender/src/slack/SlackEvent.ts

```diff
@@ -200,7 +216,7 @@ export default class SlackEvent implements Event<SlackRawEvent> {
    *
    */
   get isText(): boolean {
-    return this.isMessage;
+    return this.isMessage || !!(this._rawEvent as SlashCommandEvent).text;
```

Comment:
I wonder if we want to treat slash command as text event.
Considering `/command` and `/command some args`, why the latter should be treated as text event and the former should not?

Reply:
I agree.
It's kind of confusing the arguments string is in `event.text`